lib/attributes-to-props.js

```diff
@@ -5,6 +5,7 @@
  */
 var utilities = require('./utilities');
 var propertyConfig = require('./property-config');
+var parser = require('style-to-object');
```

Comment:
Do you mind renaming `parser` to `styleToObject` just for keeping it explicit here?